components/Tab/Tab.tsx

```diff
@@ -10,22 +10,34 @@ export const TabList: FC = ({ children, ...props }) => (
 type TabPanelProps = {
 	name: string;
 	activeTab: string;
+	fullHeight?: boolean;
 };
 
-export const TabPanel: React.FC<TabPanelProps> = memo(({ name, activeTab, children, ...props }) =>
-	activeTab === name ? (
-		<TabPanelContainer
-			id={`${name}-tabpanel`}
-			role="tabpanel"
-			aria-labelledby={`${name}-tab`}
-			tabIndex={-1}
-			{...props}
-		>
-			{children}
-		</TabPanelContainer>
-	) : null
+export const TabPanel: React.FC<TabPanelProps> = memo(
+	({ name, activeTab, fullHeight, children, ...props }) =>
+		activeTab === name ? (
+			<TabPanelContainer
+				id={`${name}-tabpanel`}
+				role="tabpanel"
+				aria-labelledby={`${name}-tab`}
+				tabIndex={-1}
+				$fullHeight={fullHeight}
+				{...props}
+			>
+				{children}
+			</TabPanelContainer>
+		) : null
 );
 
-const TabPanelContainer = styled.div`
+const TabPanelContainer = styled.div<{ $fullHeight?: boolean }>`
 	outline: none;
+	${(props) =>
+		props.$fullHeight &&
+		css`
+			flex-grow: 1;
+			& > div,
+			& > div > div {
+				height: 100%;
```

Comment:
Would be better if we could put child styling on the child div components themselves. Also as this seems to be only used in one place there's maybe no need to have some dynamic styling. I think we could just focus on styling just the container and then how the children fit in can be done on the tab components themselves.

Reply:
@avclarke I think I found a solution. However, for it to work, I'll have to make the mobile changes first. We should probably add this to the list of things to handle in a separate PR, and I'll handle it alongside the mobile changes.